app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -17,6 +17,10 @@
   <div class="govuk-grid-column-two-thirds">
     <p class="govuk-hint govuk-!-margin-bottom-8 govuk-!-margin-top-2"><%= @application_form_presenter.updated_at %></p>
 
+    <% if FeatureFlag.active?('apply_again_rejection_reasons') && @application_form.apply_2? && @previous_application_form.application_choices.rejected.present?  %>
```

Comment:
```suggestion
    <% if FeatureFlag.active?('apply_again') && @application_form.apply_2? && @previous_application_form.application_choices.rejected.present?  %>
```

Reply:
The `.apply_2?` could also become `.original_application_form.exists?` or such, later